src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -82,6 +83,48 @@
             .getName());
 
     private static OpenShiftClient openShiftClient;
+    private static String jenkinsPodNamespace = null;
+    
+    static {
+        jenkinsPodNamespace = System
+                .getProperty(Constants.OPENSHIFT_PROJECT_ENV_VAR_NAME);
+        if (jenkinsPodNamespace != null && jenkinsPodNamespace.trim().length() > 0) {
+            jenkinsPodNamespace = jenkinsPodNamespace.trim();
+        } else {
+            File f = new File(Constants.OPENSHIFT_PROJECT_FILE);
+            if (f.exists()) {
+                FileInputStream fis = null;
+                try {
+                    fis = new FileInputStream(f);
```

Comment:
might use a FileReader + ReadLine instead
https://www.caveofprogramming.com/java/java-file-reading-and-writing-files-in-java.html#readtext

Reply:
pushed